src/Microsoft.TestPlatform.Utilities/Microsoft.TestPlatform.Utilities.csproj

```diff
@@ -15,8 +15,11 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.TestPlatform.CoreUtilities\Microsoft.TestPlatform.CoreUtilities.csproj" />
     <ProjectReference Include="..\Microsoft.TestPlatform.ObjectModel\Microsoft.TestPlatform.ObjectModel.csproj" />
+    <PackageReference Include="Microsoft.VisualStudio.Coverage.CoreLib.Net">
+	    <Version>16.9.0-beta.20560.1</Version>
```

Comment:
Can you please use the latest version? 16.9.0-beta.20568.1,

Reply:
Done.